modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ListTypeDefinition/ListTypeEntriesModal.tsx

```diff
@@ -29,6 +29,7 @@ export interface IModalState extends Partial<PickListItem> {
 	pickListId?: number;
 	readOnly?: boolean;
 	reloadIframeWindow?: () => void;
+	system?: boolean;
```

Comment:
@pedro-oliveira446 Not related to your current changes, but can you please add a blank line between the lines 22 and 23 of this file? Thanks :)